pipeline/beam_tables.py

```diff
@@ -278,15 +291,21 @@ def _between_dates(filename: str,
   return True
 
 
-def _read_satellite_tags(filename: str, line: str) -> Iterator[Dict[str, Any]]:
+def _read_satellite_tags(filename: str, line: str) -> Iterator[Row]:
   """Read data for IP tagging from Satellite.
 
     Args:
       filename: source Satellite file
-      line: json str (dictionary containing tag data)
+      line: json str (dictionary containing geo tag data)
 
     Yields:
-      Processed Satellite fields
+      A row dict of the format
```

Comment:
`line` is not always geo tag data (depends on the source file). The tag output can be: 
- `{'ip': ..., 'name': ..., 'date': ...}` 
- `{'ip': ..., 'country': ..., 'date': ...}`
-
```
 { 
     'ip': ...,
     'http': ..., # optional
     'cert': ..., # optional
     'asname': ..., # optional
     'asnum': ..., # optional
     'date': ...
}
```

Reply:
updated comment and started a new [test job](https://console.cloud.corp.google.com/dataflow/jobs/us-west2/2021-07-26_15_00_02-14191324835165023980?project=firehook-censoredplanet).

Reply:
test job succeeded